Dockerfile

```diff
@@ -52,7 +52,10 @@ RUN git clone https://github.com/NVIDIA/NeMo.git && \
         git checkout FETCH_HEAD; \
     fi && \
     pip uninstall -y nemo_toolkit sacrebleu && \
-    git cherry-pick --no-commit -X theirs fa8d416793d850f4ce56bea65e1fe28cc0d092c0 a7f0bc1903493888c31436efc2452ff721fa5a67 && \
+    git cherry-pick --no-commit -X theirs \
```

Comment:
@trias702 checking if you have any objections to these commits being picked in. They look benign to me